crd/multitenantpodnetworkconfig/README.md

```diff
@@ -0,0 +1,3 @@
+# MultitenantPodNetworkConfig CRDs
+
+MPNC objects represent the network configuration goal state for a pod running a multitenant networked container and are created and managed by control plane as part of the network configuration, during Pod lifecycle events.
```

Comment:
I would also add that it is created by AKS Customers. So any change in the CRD definition has to be backward compatible.

Reply:
MPNC object will be created by the swift control plane, though - the crds created by the AKS customers will be on a separate PR. The point on backward compat is valid, but the interface is between CNS and control plane.